src/xdocs/config_blocks.xml

```diff
@@ -150,6 +150,24 @@ switch (a)
     <section name="EmptyBlock">
       <subsection name="Description">
         <p> Checks for empty blocks. This check does not validate sequential blocks. </p>
+
+        <p> This check processes LITERAL_CASE and LITERAL_DEFAULT separately: </p>
```

Comment:
please move this block below source for sequential blocks. Please show user what value should have `tokens` to have no violation.